zer0-os/zOS
# feat: implement get favorite tags for room

### What does this do?
- gets tags for a room specific to favorite tags.

### Why are we making this change?
- to return and set if a room `isFaovrite` or not.

### How do I test this?
- run tests as usual.
- alternatively, you could call the `isRoomFavorite` and check if the conversation is favorite.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

GET Favorites Demo (once wired up) with rough three dot menu

Page Refreshes - Saved State

https://github.com/zer0-os/zOS/assets/39112648/c509ba6c-b2d0-4520-8ac2-d662a69282ed
